docsets/SpigotMC/docset.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "SpigotMC",
     "version": "1.12.2",
-    "archive": "SpigotMC-1.12.2.tgz",
+    "archive": "versions/1.12.2/SpigotMC-1.12.2.tgz",
```

Comment:
This is not supported. The main docset archive needs to be separate from one in `specific_versions`.

Reply:
Alright fixed again! Sorry about the trouble, I don't do publish docsets very often.